frontend/src/container/AppLayout/index.tsx

```diff
@@ -272,11 +259,12 @@ function AppLayout(props: AppLayoutProps): JSX.Element {
 		// after logging out hide the trial expiry banner
 		if (!isLoggedIn) {
 			setShowTrialExpiryBanner(false);
+			setShowPaymentFailedWarning(false);
 		}
 	}, [isLoggedIn]);
 
 	const handleUpgrade = (): void => {
-		if (role === 'ADMIN') {
+		if (user.role === 'ADMIN') {
```

Comment:
safety check

Reply:
not required as the user.role will always be present [ default is VIEWER ]